src/add_remove_owner.test.js

```diff
@@ -19,6 +19,18 @@ import { generalInterface } from '../utils/selectors/generalInterface'
 import { rejectPendingTxs } from '../utils/actions/rejectPendingTxs'
 import { errorMsg } from '../utils/selectors/errorMsg'
 
+/*
+Add/remove Owners
+-- Enter add owner form
+-- Validate Owner name and address required, invalid address, duplicated address
+-- Input valid owner name and address,
+-- Checks them in review step and submits, confirms and executes
+-- Finds new owner in the owners list, clicks on remove owner
+-- Sets threshold value to "2"
+-- Verifies owner to be removed name an address
+-- Signs and executes. Verifies tx success status
+*/
```

Comment:
I suggest that we insert each line in the place where this action is performed. Not as one big chunk.
And not as a comment but as a console.log, so that you can see what's happening in the CI logs.

Reply:
I think that would be ok but the problem is that we would only be able to use console.logs for description and not for tracking each run(e.g.: console.loging the nonce of a tx)
If you think that's ok i'll change that and change the script

Reply:
I don't understand what you mean. I meant that the console.logs will help understand what's going on in the test when it's running on CI. But we should still extract them statically, like you did with comments.

Reply:
Ok no problem, will do that